docs/source/DeveloperDocumentation/PackagingGuides/shaka-packager.md

```diff
@@ -103,9 +101,9 @@ Please note, the `Widevine_PSSH` from the CPIX Document is in Base64 format. Sha
    --mpd_output /media/<name-of-manifest>.mpd
 ```
 
-### Packaging Content with Fairplay DRM
+### Packaging Content with FairPlay DRM
 
-Below is an example of how we can send a request to Shaka Packager to encrypt our content with Fairplay DRM and package it: 
+Below is an example of how we can send a request to Shaka Packager to encrypt our content with FairPlay DRM and package it: 
```

Comment:
```suggestion
Below is an example of how we can send a request to Shaka Packager to encrypt and package our content with FairPlay DRM. 
```
Same as above, plus changing `:` to `.` for consistency.

Reply:
Actioned.